tests/ui/privacy/restricted/private-in-public.rs

```diff
@@ -1,10 +1,14 @@
+// build-pass
```

Comment:
```suggestion
// check-pass
```
Shouldn't checking be enough? Type privacy lints are not reported *very* late.